roles/confluent.kafka_broker/tasks/health_check.yml

```diff
@@ -3,9 +3,9 @@
   shell: |
     {{ binary_base_path }}/bin/kafka-topics --bootstrap-server {{inventory_hostname}}:{{kafka_broker_listeners[kafka_broker_inter_broker_listener_name]['port']}} \
       --describe --under-replicated-partitions --command-config {{kafka_broker.client_config_file}}
-  register: topics
+  register: topics1
```

Comment:
Maybe try to name these variables better?

Reply:
Sure Erik. I am thinking
topic1 -> urp_topic
topic2 -> urp_topic_secrets_protection
WDYT, any names you'd like to suggest?

Reply:
SGTM! 👍